.travis.yml

```diff
@@ -0,0 +1,29 @@
+# Travis configuration file using the build matrix feature
+# Read more under http://docs.travis-ci.com/user/build-configuration/
+# THIS SCRIPT IS SUPPOSED TO BE AN EXAMPLE. MODIFY IT ACCORDING TO YOUR NEEDS!
+
+sudo: false
+language: python
+virtualenv:
+  system_site_packages: true
+env:
+  matrix:
+    - DISTRIB="conda" PYTHON_VERSION="3.3" COVERAGE="true"
+    - DISTRIB="conda" PYTHON_VERSION="3.4" COVERAGE="false"
+addons:
+  apt:
+    packages:
+      - git
+      - python-pip
+install:
+  - source tests/travis_install.sh
```

Comment:
I've also added a number of lines of docstrings for the documentation.  One file counts as new lines because I corrected the indentation etc.